code/__DEFINES/preferences.dm

```diff
@@ -43,4 +43,5 @@
 #define BE_GANG			4096
 #define BE_SHADOWLING	8192
 #define BE_ABDUCTOR		16384
-#define BE_REVENANT		32768
+#define BE_REVENANT		32768
+#define BE_CLONE		65536

```

Comment:
someone said that the flag could store values that high, but you wouldn't be able to do operations on them. we'll see ayy lmao